src/common/Modal.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ReactNode, Component } from 'react';
+import { Modal, Title, TitleLevel, BaseSizes } from '@patternfly/react-core';
+
+interface Props {
+  open: boolean;
+  close: () => void;
+}
+
+export class DialogModal extends Component<Props> {
+  render(): React.ReactNode {
+    return (
+      <Modal
+        isLarge
+        hideTitle={true}
+        title="is hidden"
+        isOpen={this.props.open}
+        onClose={this.props.close}
+      >
+        {this.props.children}
```

Comment:
This component doesn't give much added value compared to the `Modal` component itself.
I believe we can remove this and put it inside the `DeleteApplicationPage` component directly